test/util_test.dart

```diff
@@ -591,8 +591,8 @@ void overReactExample() {}''';
 
       test('returns empty list when input has no descendants', () {
         final node = parseAndGetSingle('''
-          UiFactory<FooProps> Foo = castUiFactory(_\$Foo); // ignore: undefined_identifier
-        ''').variables.variables.first.root;
+          var foo = bar;
+        ''').variables.variables.first.initializer!;
```

Comment:
I changed this since it was using the `NamedType` `Foo` since it isn't visited anymore (see https://github.com/Workiva/over_react_codemod/pull/249/files#r1346598729), and swapped this with a simpler snippet so it was easier to reason about. Now, `node` is `bar` by using `.initializer`.